composer.json

```diff
@@ -187,14 +190,17 @@
             "acquia/blt": {
                 "NCIOCPL#48": "patches/blt/issue-48-fix-behat-chrome-invocation.patch"
             },
-            "drupal/pathauto":{
-                "3003373 : Wrong verbose messenger method" : "https://www.drupal.org/files/issues/2018-09-30/3003373-2.patch"
+            "drupal/pathauto": {
+                "3003373 : Wrong verbose messenger method": "https://www.drupal.org/files/issues/2018-09-30/3003373-2.patch"
             },
-            "drupal/address":{
+            "drupal/address": {
                 "2856797: Missing theme suggestions for address_plain": "https://www.drupal.org/files/issues/2018-04-10/address-missing_theme_suggestions_for_address_plain-2856797-10.patch"
             },
-            "drupal/token_filter":{
+            "drupal/token_filter": {
                 "3036541: Missing token_filter schema file": "https://www.drupal.org/files/issues/2019-03-04/3036541-2-token_filter_schema.patch"
+            },
+            "drupal/simplesamlphp_auth": {
+                "2907182: Admin UI Pages not Accessible via Permission": "https://www.drupal.org/files/issues/simplesamlphp_auth-admin_ui_pages-2907182-30.patch"
             }
         },
         "drupal-scaffold": {
```

Comment:
@blairlearn I see you have a patch for the .htaccess changes but its not in your composer file. Can we add it in here? This can go under the "scripts:" section. Example below (and recommend creating separate patches for each set of changes required):

```
"scripts": {
        "post-drupal-scaffold-cmd": [
             "cd docroot && patch -p1 --forward <../patches/htaccess-ssl.patch",
            "cd docroot && patch -p1 --forward <../patches/htaccess-saml.patch",
            "cd docroot && patch -p1 --forward <../patches/htaccess-hsts.patch"
        ],
    ...
```

Reply:
The SAML patch is already there.  There's nothing for SSL or HSTS (yet), but they **must** not be applied that way. Since the ODEs and local machines will likely never have SSL (not without a great deal of ongoing pain and suffering). Current plan is to apply them conditionally via BLT invoked from the same spot. (See #1462 for the lowdown.)